robot/EDA/resources/EducationCloudSettingsPageObject.py

```diff
@@ -37,4 +37,13 @@ def click_app_in_edc_home(self,value):
         self.selenium.wait_until_page_contains_element(locator, timeout=60, error=f'{locator} is not available')
         self.selenium.wait_until_element_is_visible(locator,
                                                 error= "Element is not displayed for the user")
-        self.selenium.click_element(locator)
\ No newline at end of file
+        self.selenium.click_element(locator)
+
+    def verify_error_is_displayed(self,value):
```

Comment:
This should work. The other option you can do is by passing the `value` to the locator and see if the browser can find it. I think most of other methods do that but it is really up to you.

Reply:
Updated to match similar methods that pass the value to the locator for the browser to see if it's on the page and visible. Thanks!